.goreleaser.develop.yaml

```diff
@@ -64,8 +64,8 @@ dockers:
     use: buildx
     goos: linux
     goarch: amd64
-    extra_files:
-      - tmp/linux_amd64/libs
+    # extra_files:
```

Comment:
Can we add a comment on why we would want to uncomment these in the future?

Reply:
added